docsets/DraftJS/docset.json

```diff
@@ -1,11 +1,27 @@
 {
     "name": "Draft.js",
-    "version": "0.6.0/2016-04-27_20:52:21",
-    "archive": "Draft.js.tgz",
+    "version": "0.10.0",
+    "archive": "DraftJS.tgz",
     "author": {
-        "name": "Remko Tronçon",
-        "link": "https://el-tramo.be"
+        "name": "Michael Sharer",
+        "link": "http://29degre.es"
     },
-    "aliases": ["draft", "draftjs"],
-    "specific_versions": []
+    "aliases": [
+        "Draft",
+        "DraftJS",
+        "React",
+        "WYSIWYG",
+        "RichText",
+        "Editor"
+      ],
+    "specific_versions": [
+        {
+          "version": "0.10.0",
+          "archive": "versions/0.10.0/DraftJS.tgz"
+        },
+        {
+          "version": "0.1.0",
+          "archive": "versions/0.1.0/DraftJS.tgz"
```

Comment:
Archive 0.1.0? If you pull down the repo as is right now, there is a directory for versions/0.1.0
I am not sure what happened along the way but there was an addition of version 0.6.0/... and changes the doc set name to Draft.js.tgz from DraftJS.tgz. Do you want me to add the version/0.6.0... archive?